### DISCUSSION TITLE
How to use tiptap in vue3.0？

### DISCUSSION THREAD

**Discussion by yangshengqian0810** [ueberdosis/tiptap#855] [General Questions]
How to use tiptap in vue3.0? 
Does tiptap have a way to adapt to vue3.0?

**Comment by hanspagel** [comment]
tiptap 2 will be compatible with Vue.js 3, but that’s not released yet. 

Follow the progress here: https://github.com/ueberdosis/tiptap/issues/547

Hope that helps!

**Reply by yangshengqian0810** [reply]
thanks @hanspagel
